lib/aws/active_job/sqs/cli.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+module Aws
+  module ActiveJob
+    module SQS
+      # Utilities for the aws_active_job_sqs CLI.
+      # @api private
+      module CLI
```

Comment:
Should this maybe be a class, taking ARGV and storing state of "use rails"?

Reply:
converted to a class